metadata.yaml

```diff
@@ -4,7 +4,7 @@ name: github-actions-exporter
 display-name: GitHub Actions Exporter
 summary: GitHub Actions Exporter
 description: Collects metrics from GitHub Actions.
-docs: ""
+docs: "https://github.com/canonical/github-actions-exporter-operator/"
```

Comment:
Shouldn't this be a link to discourse?

Reply:
GitHub Actions is a WIP/internal charm yet so there is no discourse documentation.